services-custom/dynamodb-enhanced/src/test/java/software/amazon/awssdk/enhanced/dynamodb/model/ScanEnhancedRequestTest.java

```diff
@@ -40,7 +43,7 @@ public void builder_minimal() {
         assertThat(builtObject.exclusiveStartKey(), is(nullValue()));
         assertThat(builtObject.consistentRead(), is(nullValue()));
         assertThat(builtObject.filterExpression(), is(nullValue()));
-        assertThat(builtObject.projectionExpression(), is(nullValue()));
+        assertThat(builtObject.attributesToProject(), is(emptyIterable()));
```

Comment:
A default optional list should be 'null' so we can differentiate between 'they didn't mention it' and 'they explicitly gave us an empty list as a value' if we ever need to.